misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/BTestRunner.java

```diff
@@ -404,6 +413,13 @@ private void executeFunction(Test test, TestSuite suite, String packageName, Cla
                                  Scheduler scheduler, AtomicBoolean shouldSkip, AtomicBoolean shouldSkipTest,
                                  List<String> failedOrSkippedTests, List<String> failedAfterFuncTests) {
         TesterinaResult functionResult;
+        Path sourceRootPath = Paths.get(suite.getSourceRootPath());
+        Path jsonCacheDir = sourceRootPath.resolve("target").resolve("caches").resolve("json_cache");
+        Path rerunJson =
+                jsonCacheDir.resolve(suite.getOrgName()).resolve(suite.getPackageID()).resolve(suite.getVersion());
+        Path jsonPath = Paths.get(rerunJson.toString(), "retun_test.json");
```

Comment:
Shall we correct the typo in "retun_test.json"

Reply:
Corrected typo in all instances.